dev/tests/static/testsuite/Magento/Test/Php/LiveCodeTest.php

```diff
@@ -68,17 +68,22 @@ private static function getChangedFilesBaseDir()
      * @param array $fileTypes
      * @param string $changedFilesBaseDir
      * @param string $baseFilesFolder
+     * @param string $whitelistFile
      * @return array
      */
-    public static function getWhitelist($fileTypes = ['php'], $changedFilesBaseDir = '', $baseFilesFolder = '')
-    {
+    public static function getWhitelist(
```

Comment:
Updates to this test are not reflected in PR description.
Please update with relevant information

Reply:
Updated.